docs/api.rst

```diff
@@ -0,0 +1,58 @@
+=============
+API Reference
+=============
+
+.. automodule:: vsutil
+
+
+Functions that return a clip
+============================
```

Comment:
We might want to think about a check that each new public function is also added here (because I feel like I’m going to forget that at some point), but that’s probably out of scope for this PR. Should be doable with just `grep` alone.

Reply:
Well, this would be doable with Sphinx via the auto-module, but the way we export everything from the submodules directly means this isn't really feasible without the order being weird, and there being prefixes that I'm not sure we even want to encourage as the structure could still change. i.e. the automodule will say `vsutil.info.get_w` instead of `vsutil.get_w` (I can double check this again but pretty sure).

Reply:
Then automodule might not be the right tool here. If this ever becomes an issue, I’ll just write a small shell script that greps through the source files and the docs and add that to the CI to make sure we didn’t forget anything.